### PR TITLE
[echarts] minor corrections to sankey series

### PR DESCRIPTION
**Pull request by softwords**
Please fill in this template.

- [x ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [ x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [ x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/apache/incubator-echarts/issues/12763
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header. All modified features are in 4.6.0

### CODE REVIEW

**Review by trajnisz — APPROVED** [review]

File: types/echarts/options/series/sankey.d.ts [hunk 1]
```diff
@@ -51,6 +51,15 @@ declare namespace echarts {
...
             id?: string;
 
+            /**
+             * Series name used for displaying in
+             * [tooltip](https://echarts.apache.org/en/option.html#tooltip)
+             *
+             *
+             * @see https://echarts.apache.org/en/option.html#series-sankey.name
+             */
+            name?: string;
```
Comment (Ovilia) [inline types/echarts/options/series/sankey.d.ts:61]: I didn't see `name` under series. If you mean the name under `data`, it shouldn't be placed here. https://echarts.apache.org/en/option.html#series-sankey.data.name .
I checked with the demo but it doesn't seem to work with tooltip.

File: types/echarts/options/series/sankey.d.ts [hunk 2]
```diff
@@ -183,6 +192,17 @@ declare namespace echarts {
...
+            /**
+             * Alignment of nodes in the graph.
+             *
+             * May be 'left', 'right' or 'justify'
+             *
+             * @default
+             * justify
+             * @see https://echarts.apache.org/en/option.html#series-sankey.nodeAlign
+             */
+            nodeAlign?: string;
```
Comment (Ovilia) [inline types/echarts/options/series/sankey.d.ts:204]: `nodeAlign` is also not available in the doc. But I tried with the demo and it works. Maybe you should make another PR to the doc: https://github.com/apache/incubator-echarts-doc

**Review by Ovilia — CHANGES_REQUESTED** [review]

File: types/echarts/options/series/sankey.d.ts (hunk 2 above)
Comment (softwords) [reply types/echarts/options/series/sankey.d.ts:204]: i have added `name` and `nodeAlign` to the documentation  to support these links in the typings file:

https://github.com/apache/incubator-echarts-doc/pull/107

`name` under series will appear in the tooltip parameters as `seriesName`, same as everywhere else:, and also in the legend, if you add one: this small change to the sankey example demonstrates:

```
option = {
    tooltip: {
        formatter: (params) => params.seriesName
    },
    legend: {},
    series: [{
        type: 'sankey',
        layout: 'none',
        name: "flow_diagram",
        focusNodeAdjacency: 'allEdges',
      
        nodeAlign: 'left',
        data: [{
            name: 'a'
        }, {
            name: 'b'
        }, {
            name: 'a1'
        }, {
            name: 'a2'
        }, {
            name: 'b1'
        }, {
            name: 'c'
        }],
        links: [{
            source: 'a',
            target: 'a1',
            value: 5
        }, {
            source: 'a',
            target: 'a2',
            value: 3
        }, {
            source: 'b',
            target: 'b1',
            value: 8
        }, {
            source: 'a',
            target: 'b1',
            value: 3
        }, {
            source: 'b1',
            target: 'a1',
            value: 1
        }, {
            source: 'b1',
            target: 'c',
            value: 2
        }]
    }]
    
};
```

**Review by Ovilia — APPROVED** [review]

**Comment by softwords** [comment]
Thanks @Ovilia for reviewing this and for your assistance. 

As I mentioned elsewhere, because every series type will support a tooltip, and tooltip parameters will always include the series name if available, I think that `name` should be present in every series definition in typescript. It is not required of course, but should not give a compile error in typescript when used. 

So, if you are happy with this, what I would like to do now is:

- make any pull requests in the echarts documentation to ensure that `name` is present for each series

- make the corresponding changes to the index.d.ts, which can then reference the documentation links

- add a simple `echarts-test.js` to verify that `id`, `type` and `name` can be used with any series type.

Please let me know if you are happy for me to proceed on these small things. I mention @plainheart here who has been following on the documentation project.

**Comment by Ovilia** [comment]
@softwords That sounds great! Thank you!

**Comment by Ovilia** [comment]
@typescript-bot Please merge this PR. I think the "Revision needed" label should be revoked.

**Comment by elibarzilay** [comment]
(It wasn't merging it because of the config file change...)
